### PR TITLE
Add bool for to elixir

### PR DESCRIPTION
**Pull request by joshuataylor**
When trying to read a bool, an error is thrown that "Bool" does not exist.

Not sure where tests should live, and if there should be tests for the different types Elixir has out of the box?

### CODE REVIEW

**Comment by sofakingworld** [comment]
@joshuataylor Hi!

Provide some tests please, for example https://github.com/balance-platform/pillar/blob/master/test/pillar_test.exs#L588 

I'm not sure, that Bool type exists in Clickhouse v20, so in this test workaround is provided

**Review by sofakingworld — CHANGES_REQUESTED** [review]
https://github.com/balance-platform/pillar/pull/49#issuecomment-1248157276

**Comment by sofakingworld** [comment]
@joshuataylor Hi again!
Are you planning to finish this development?

**Comment by joshuataylor** [comment]
Yes! Sorry for the delay on this, I will get this done ASAP

**Comment by joshuataylor** [comment]
So a very annoying thing with the new API:

We can't use strings for true/false, we have to send 1/0 or true/false, and not strings.

Sending string values gives back this error:
```elixir
              %Pillar.HttpClient.Response{status_code: 500, body: "Code: 467. DB::Exception: Invalid boolean value, should be true/false, 1/0.: (while reading the value of key field5): (at row 1)\n: While executing ParallelParsingBlockInputFormat. (CANNOT_PARSE_BOOL) (version 22.5.4.19 (official build))\n", headers: [{"date", "Fri, 23 Sep 2022 04:42:10 GMT"}, {"connection", "Keep-Alive"}, {"content-type", "text/plain; charset=UTF-8"}, {"x-clickhouse-server-display-name", "b711072987f3"}, {"transfer-encoding", "chunked"}, {"x-clickhouse-exception-code", "467"}, {"keep-alive", "timeout=3"}, {"x-clickhouse-summary", "{\"read_rows\":\"0\",\"read_bytes\":\"0\",\"written_rows\":\"0\",\"written_bytes\":\"0\",\"total_rows_to_read\":\"0\"}"}]}
```

I've also bumped the CH version in the github YAML.

File: .github/workflows/elixir.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ jobs:
       matrix:
         otp: [23.1.2]
         elixir: [1.10.4, 1.11.4, 1.12.0, 1.13.4]
-        clickhouse: [20.3.5.21, 21.9.6.24, 22.1.3.7]
+        clickhouse: [20.3.5.21, 21.9.6.24, 22.1.3.7, 22.9.1.2603]
```
Comment (sofakingworld) [inline .github/workflows/elixir.yml:27]: 22.9.1.2603 doesn't exist on dockerhub, so i remove this

**Comment by sofakingworld** [comment]
Digits works, so i added 0/1 for true/false.

Thanks for PR and issue #50 

I'will make some workarounds and release new version

**Comment by joshuataylor** [comment]
Cheers!

**Pull request merged**